.eslintrc.json

```diff
@@ -4,6 +4,7 @@
         "html"
     ],
     "rules": {
+        "comma-dangle": ["error", "never"],
```

Comment:
I'm surprised this wasn't already covered, we probably want this to be part of https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Tools/eslint-config-cesium/index.js instead of specific to Cesium, right?